openssh/config_ini.sls

```diff
@@ -13,7 +13,7 @@ sshd_config-with-ini:
   {%- if salt['file.contains_regex'](openssh.sshd_config, regex_search_for_tabs) %}
   file.replace:
     - name: {{ openssh.sshd_config }}
-    - pattern: {{ regex_search_for_tabs }}
+    - pattern: '{{ regex_search_for_tabs }}'
```

Comment:
Good catch! But this implementation does not account for single quotes in the regular expression.

```suggestion
    - pattern: {{ regex_search_for_tabs | yaml_dquote }}
```
As documented in https://docs.saltproject.io/en/latest/topics/jinja/index.html#yaml-dquote.

Does this work for you? If it does, please update this PR. I'll merge it then.

Reply:
Hello @alxwr 

I got the same issue today and your suggestion is working properly

Cheers